collabora_online.module

```diff
@@ -116,15 +118,23 @@ function collabora_online_entity_operation(EntityInterface $entity): array {
     ],
   ];
```

Comment:
To make this function complete, we should also check for viewability, not just editability.
However we would have to implement the fallback logic from 'view' to 'edit' and 'view_comment', to get a desirable outcome with Collabora.